### ISSUE TITLE
Account for duplicate empty names in blender when exporting

### ISSUE THREAD

**Issue by MadmanLava** [ross-g/io_pdx_mesh#53]
In blender, when making Stellaris ships, you currently have to make a separate blend file for EVERY individual section so as to avoid duplicate empty names, as well as the previously submitted issue where you cannot specify locators to export, as it will always export every locator in the file.

Assuming that was fixed, if the plugin could also remove the extra ".00#" added to the names of duplicates when exporting empties, it would handily solve locator name conflicts and thus allow the creation and export of multiple entities within the comfort of the same blender file instead of having to make several dozen(in the case of a Stellaris shipset)

**Comment by ross-g** [comment]
Interesting... so then is your idea to have a single scene with all of the meshes/empties and an armature in it, that would export to multiple modular ship parts (with _export selected_ working for empties) ?

And currently one of the things stopping that from being possible is that Blender does not allow duplicate names in the same scene?

**Comment by MadmanLava** [comment]
I didnt really mean exporting to multiple ship sections _at the same time_ so much as I meant just being able to work on them within the same .blend file, as well as individually export them and their locators from the same file without hassle.

The problem is when exporting something with locators its really hard to get blender to rename the specific locators you want as the "originals" without the ".00#" so they work, and even then every other empty in the file gets dragged along anyways regardless of your selection which is messy and can cause problems depending on what you're exporting.

TL;DR:
- Please make the exporter only export _selected_ locators(when applicable)
- Please make the exporter account for empties blender considers "duplicates", only saving the actual locator name to the .mesh file and not the number it appends, so that the user doesnt have to work around this.

Sorry if I wasnt very clear the first time, Im not great at being both concise and specific, but this would be an absolute lifesaver for me!
